torch/testing/_internal/common_distributed.py

```diff
@@ -394,6 +418,18 @@ def run_test(self, test_name, pipe):
             pipe.close()
             sys.exit(MultiProcessTestCase.TEST_ERROR_EXIT_CODE)
 
+    def _get_timedout_process_traceback(self):
+        for i, process in enumerate(self.processes):
+            if process.exitcode is None:
+                pipe = self.pid_to_pipe[process.pid]
+                pipe.send(MultiProcessTestCase.Event.GET_TRACEBACK)
```

Comment:
nit: send out all the requests before waiting for each proc so they can be processed in parallel by the subprocesses might be better? Might be overkill for this though